rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -124,6 +124,15 @@ def __init__(self):
                 print("--address argument provided without a value.")
                 sys.exit(-1)
 
+        url_path = self.declare_parameter("url_path", "/").value
+        if "--url_path" in sys.argv:
+            idx = sys.argv.index("--url_path") + 1
+            if idx < len(sys.argv):
+                url_path = str(sys.argv[idx])
+            else:
+                print("--url_path argument provided without a value.")
+                sys.exit(-1)
```

Comment:
I see that this is copied from above. Though I think this code is sensitive for the following error:
`--option1 --option2 value2`.
So it might be good to change this in another PR.

Reply:
Is there an accepted and typical args parsing library that python projects use or should be using? I'd say its outside the scope of this PR to switch but it does seem like it would avoid hand-crafted args parsing like what is happening here.

Reply:
https://github.com/docopt/docopt looks pretty nice. I wonder how hard it would be to use it while still keeping the CLI backwards compatible with the current implementation.

Reply:
How about the standard [argparse module](https://docs.python.org/3/library/argparse.html)?